README.md

```diff
@@ -1,14 +1,16 @@
-# Pusher HTTP Go Library
+# Pusher Channels HTTP Go Library
 
 [![Build Status](https://travis-ci.org/pusher/pusher-http-go.svg?branch=master)](https://travis-ci.org/pusher/pusher-http-go) [![Coverage Status](https://coveralls.io/repos/pusher/pusher-http-go/badge.svg?branch=master)](https://coveralls.io/r/pusher/pusher-http-go?branch=master) [![GoDoc](https://godoc.org/github.com/pusher/pusher-http-go?status.svg)](https://godoc.org/github.com/pusher/pusher-http-go)
 
-The Golang library for interacting with the Pusher HTTP API.
+The Golang library for interacting with the Pusher Channels HTTP API.
 
-This package lets you trigger events to your client and query the state of your Pusher channels. When used with a server, you can validate Pusher webhooks and authenticate private- or presence-channels.
+This package lets you trigger events to your client and query the state of your Pusher channels. When used with a server, you can validate Pusher Channels webhooks and authenticate `private-` or `presence-` channels.
 
-In order to use this library, you need to have a free account on <http://pusher.com>. After registering, you will need the application credentials for your app.
+Register for free at <https://pusher.com> and use the application credentials within your app as shown below.
```

Comment:
I'd modify this link to point to https://pusher.com/channels - just to make it more specific.

Reply:
👍